README.md

```diff
@@ -437,6 +437,20 @@ or any other nested information.
 
 <img width="500" alt="A refactored DAG showing three sources which are each being referenced by an accompanying staging model" src="https://user-images.githubusercontent.com/30663534/159603703-6e94b00b-07d1-4f47-89df-8e5685d9fcf0.png"> 
 
+### Exposure Materializations
+#### Model
+
+`fct_exposure_materializations` ([source](models/marts/dag/fct_exposure_materializations.sql)) shows each model with a direct relationship to an exposure that is *not* materialized as a table in the warehouse. 
```

Comment:
I _do_ have opinions about the model name lol - I think this is a bit confusing because it makes me think it's related to the materialization of the actual exposure (which doesn't make sense). How about something like `fct_exposure_parents_materialization`?

Reply:
i'm cool with that!

Reply:
Awesome! Looks like you already updated the model names but still need to update the readme to match!

Reply:
updated reference in README!